app/views/admin/document_collection_email_subscriptions/edit.html.erb

```diff
@@ -18,12 +18,6 @@
     } %>
 
     <h2 class="govuk-heading-l">Email notifications</h2>
-
-    <% if taxonomy_topic_cannot_be_set?(@collection) %>
-      <%= render partial: "email_override_summary_page" %>
-    <% else %>
-      <p class="govuk-body">Choose the type of email updates users will get if they sign up for notifications.</p>
-      <%= render partial: "form" %>
-    <% end %>
+    <%= render partial: "email_override_summary_page" %>
```

Comment:
Shall we simplify this template? 
- We could lose lines 4-11 as we won't ever have errors
- We could move the content from the email override summary page into this partial, given it's now so simple.